core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1000,6 +1000,13 @@ public ServerBuilder annotatedService(String pathPrefix, Object service,
         return this;
     }
 
+    /**
+     * Returns a {@link AnnotatedServiceBindingBuilder} to build annotated service.
+     */
+    public AnnotatedServiceBindingBuilder annotatedService() {
```

Comment:
Could you add this method to `VirtualHostBuilder` as well?

Reply:
Sure will do.